galaxy_release_util/bootstrap_history.py

```diff
@@ -309,7 +308,6 @@
 
 - [ ] **Deploy and Test Release on galaxy-main**
     - [ ] Update usegalaxy.org to ensure it is running the ``release_${version}`` branch.
-    - [ ] Deploy to toolshed.g2.bx.psu.edu.
```

Comment:
Maybe keep, but rephrase as "Once we decide to migrate the ToolShed to v2.0, ..." ?

Reply:
same as my comment above.